runtime/executor/src/main/java/org/apache/nemo/runtime/executor/task/MultiThreadParentTaskDataFetcher.java

```diff
@@ -139,9 +139,9 @@ private void fetchDataLazily() {
           LOG.error(exception.getMessage());
           throw new RuntimeException(exception);
         }
-      });
+      })
```

Comment:
Could you merge the lines 142-144 together, so that it becomes something like `})));` ?
Other code follow the convention, so it would be better to keep it that way. Thanks!